### PR TITLE
Mugen/fix bullets

### PR DESCRIPTION
**Pull request by 0xMugen**

### CODE REVIEW

File: refactor/.gitignore [hunk 1]
```diff
@@ -19,3 +19,4 @@ Thumbs.db
 # Vite
 vite.config.js.timestamp-*
 vite.config.ts.timestamp-*
+bun.lockb
```
Comment (knownasred) [inline refactor/.gitignore:22]: This is required for cloudflare building

File: refactor/src/dojoConfig.ts [hunk 2]
```diff
@@ -4,6 +4,7 @@ import { createDojoConfig } from '@dojoengine/core'
 
 export const dojoConfig = createDojoConfig({
   manifest,
+  toriiUrl: 'http://localhost:8080',
```
Comment (knownasred) [inline refactor/src/dojoConfig.ts:7]: This will break the deployment. Is it possible to add it in an environment?

**Review by knownasred — CHANGES_REQUESTED** [review]
Small changes to make sure this will work on the deployed version

File: refactor/.gitignore (hunk 1 above)
Comment (0xMugen) [reply refactor/.gitignore]: readded

File: refactor/src/dojoConfig.ts (hunk 2 above)
Comment (knownasred) [reply refactor/src/dojoConfig.ts]: 2fast4me

**Review by knownasred — APPROVED** [review]
LGTM

**Pull request merged**